pkg/config/example_test.go

```diff
@@ -1417,3 +1417,399 @@ func ExampleNewCreateChannelTx() {
+	// 						"policy": {
+	// 							"type": 3,
+	// 							"value": {
+	// 								"rule": "ANY",
+	// 								"sub_policy": "Writers"
+	// 							}
+	// 						},
+	// 						"version": "0"
+	// 					}
+	// 				},
+	// 				"values": {
+	// 					"BatchSize": {
+	// 						"mod_policy": "",
+	// 						"value": {
+	// 							"absolute_max_bytes": 100,
+	// 							"max_message_count": 100,
+	// 							"preferred_max_bytes": 100
+	// 						},
+	// 						"version": "0"
+	// 					},
+	// 					"BatchTimeout": {
+	// 						"mod_policy": "",
+	// 						"value": {
+	// 							"timeout": "15s"
+	// 						},
+	// 						"version": "0"
+	// 					},
+	// 					"Capabilities": {
+	// 						"mod_policy": "Admins",
+	// 						"value": {
+	// 							"capabilities": {
+	// 								"V1_3": {}
+	// 							}
+	// 						},
+	// 						"version": "0"
+	// 					},
+	// 					"ChannelRestrictions": {
+	// 						"mod_policy": "Admins",
+	// 						"value": {
+	// 							"max_count": "1"
+	// 						},
+	// 						"version": "0"
+	// 					},
+	// 					"ConsensusType": {
+	// 						"mod_policy": "Admins",
+	// 						"value": {
+	// 							"metadata": null,
+	// 							"state": "STATE_NORMAL",
+	// 							"type": "kafka"
+	// 						},
+	// 						"version": "0"
+	// 					},
+	// 					"KafkaBrokers": {
+	// 						"mod_policy": "Admins",
+	// 						"value": {
+	// 							"brokers": [
+	// 								"kafka0:9092",
+	// 								"kafka1:9092"
+	// 							]
+	// 						},
+	// 						"version": "0"
+	// 					}
+	// 				},
+	// 				"version": "1"
+	// 			}
+	// 		},
+	// 		"mod_policy": "",
+	// 		"policies": {
+	// 			"Admins": {
+	// 				"mod_policy": "Admins",
+	// 				"policy": {
+	// 					"type": 3,
+	// 					"value": {
+	// 						"rule": "MAJORITY",
+	// 						"sub_policy": "Admins"
+	// 					}
+	// 				},
+	// 				"version": "0"
+	// 			},
+	// 			"Readers": {
+	// 				"mod_policy": "Admins",
+	// 				"policy": {
+	// 					"type": 3,
+	// 					"value": {
+	// 						"rule": "ANY",
+	// 						"sub_policy": "Readers"
+	// 					}
+	// 				},
+	// 				"version": "0"
+	// 			},
+	// 			"Writers": {
+	// 				"mod_policy": "Admins",
+	// 				"policy": {
+	// 					"type": 3,
+	// 					"value": {
+	// 						"rule": "ANY",
+	// 						"sub_policy": "Writers"
+	// 					}
+	// 				},
+	// 				"version": "0"
+	// 			}
+	// 		},
+	// 		"values": {
+	// 			"Capabilities": {
+	// 				"mod_policy": "Admins",
+	// 				"value": {
+	// 					"capabilities": {
+	// 						"V1_3": {}
+	// 					}
+	// 				},
+	// 				"version": "0"
+	// 			},
+	// 			"OrdererAddresses": {
+	// 				"mod_policy": "Admins",
+	// 				"value": {
+	// 					"addresses": [
+	// 						"127.0.0.1:7050"
+	// 					]
+	// 				},
+	// 				"version": "0"
+	// 			}
+	// 		},
+	// 		"version": "0"
+	// 	},
+	// 	"sequence": "0"
+	// }
+}
+
+func ExampleConfigTx_AddOrdererCapability() {
+	baseConfig := fetchChannelConfig()
+	c := config.New(baseConfig)
+
+	err := c.AddOrdererCapability("V1_4")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Not sure if you wanted to add the protolator output here and the ones below

Reply:
I did not add the output intentionally - given how little value it adds.